setup.py

```diff
@@ -57,6 +57,11 @@
             'djangorestframework',
             'django-multiselectfield',
             'django-crispy-forms',
+            'python-memcached',
```

Comment:
I'd remove this since we should not push a caching solution onto another developer.